recipes-nemomobile/sensorfw/sensorfw-hybris-hal-plugins_git.bb

```diff
@@ -0,0 +1,18 @@
+require sensorfw.inc
+
+EXTRA_QMAKEVARS_PRE = "CONFIG+=autohybris"
+
+do_install:append() {
+    # Remove sensorfw core elements.
+    rm -rvf ${D}/usr/sbin/
+    rm -rvf ${D}/usr/bin/
+    rm -rvf ${D}/lib/
+    rm -rvf ${D}/etc/
+    rm -rvf ${D}/usr/share/
+    rm -rvf ${D}/usr/lib/libsensor*
+    rm -rvf ${D}/usr/lib/pkgconfig/
+}
+
+DEPENDS += "sensorfw libhybris"
+
+FILES:${PN} = "/usr/lib /usr/lib/sensord-qt5/*.so /usr/lib/sensord-qt5/testing/*.so"
```

Comment:
Is that necessary ? I expect all the content of /usr (that is, all the content listed here) would end up in the right recipe ?

Reply:
Good point! `/usr/lib` isn't required here.